### PR TITLE
Fix #175: Broken phpMyAdmin version string stripping

### LINKED ISSUE #175: Version string cleanup not working as desired
It seems that new reports still get full versions instead of stripped ones, for example see https://reports.phpmyadmin.net/reports/view/34390.

### PR DESCRIPTION
**Pull request by devenbansod**
Strip `pma_version` correctly while creating a new report too

Fix #175 
Also,
* Add related tests to IncidentsControllerTest
* Fix json formatting of incidents

Signed-off-by: Deven Bansod <devenbansod.bits@gmail.com>

### CODE REVIEW

**Comment by devenbansod** [comment]
We might have to run the migration again to take care of the reports added between this period.

**Comment by nijel** [comment]
Do you know how to rerun specific migration (without running others)?

File: src/Model/Table/IncidentsTable.php [hunk 1]
```diff
@@ -309,6 +309,7 @@ protected function _getReportDetails($bugReport, $index = 0)
                 'location' => $location,
                 'linenumber' => is_null($linenumber) ? 0 : $linenumber,
                 'pma_version' => $bugReport['pma_version'],
+                'pma_version' => $this->getStrippedPmaVersion($bugReport['pma_version']),
```
Comment (nijel) [inline src/Model/Table/IncidentsTable.php]: The 'pma_version' shouldn't be there twice...

**Comment by devenbansod** [comment]
I could not find a proper way easily.

So I used to do the following:
In the Error reporting server's database, run:
`mysql > DELETE FROM phinxlog WHERE migration_name = 'FixLongPmaVersions'`
and then run
`$ bin/cake migrations migrate -t FixLongPmaVersions`

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**